WordPress/gutenberg
# Update the backup icon to better align with WordPress icon package dna

I believe the current backup icon – used in the "Revisions" link – is based on an old Dashicon. This PR updates it to match the other icons in the WordPress Icons package, namely:

* The stroke on the arrow is now 1.5px
* Arrow head matches the "rotate" icons that already exist

Before:

<img width="298" alt="Screenshot 2020-12-03 at 11 14 22" src="https://user-images.githubusercontent.com/846565/101013002-900d0300-355b-11eb-9166-2175353e7349.png">

After:

<img width="289" alt="Screenshot 2020-12-03 at 11 32 17" src="https://user-images.githubusercontent.com/846565/101013020-9602e400-355b-11eb-91a7-b5bc44fc8f2b.png">

Rotate icon for reference:

<img width="198" alt="Screenshot 2020-12-03 at 11 35 20" src="https://user-images.githubusercontent.com/846565/101013076-a74bf080-355b-11eb-8c85-38a26c40ec5e.png">

Additional considerations:

The new icon is a 1:1 match with the rotate icon, just rotated slightly to make room for the "hands" of the "clock". If we break that 1:1 relationship ever so slightly we get a sharper icon.

So there's a question to be asked around sharpness vs consistency. Thoughts?

<img width="627" alt="Screenshot 2020-12-03 at 12 29 16" src="https://user-images.githubusercontent.com/846565/101018299-34df0e80-3563-11eb-8fe5-e861b8f93c0b.png">

## To test:

* Open a post and check the revisions link in the sidebar. It should look like that "After" screenshot above.
